README.md

```diff
@@ -26,6 +26,8 @@ but relies on the default scheduler for that.
 
 ## Build and Run
 
+- Checkout the repo into your $GOPATH directory under src/sigs.k8s.io/descheduler
```

Comment:
I don't think that's necessary, given the current state of art with go, and especially go.mod. Can be removed in a followup.